# Czech translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2012-06-14 00:53+0100\n"
"PO-Revision-Date: 2012-03-21 06:14+0000\n"
"Last-Translator: Vojtěch Trefný <vojtech.trefny@gmail.com>\n"
"Language-Team: Czech <cs@li.org>\n"
"Language: cs\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2012-04-17 13:40+0000\n"
"X-Generator: Launchpad (build 15099)\n"

#. TRANSLATORS: download size of small updates, e.g. "250 kB"
#: ../DistUpgrade/utils.py:433 ../UpdateManager/Core/utils.py:433
#, python-format
msgid "%(size).0f kB"
msgid_plural "%(size).0f kB"
msgstr[0] "%(size).0f kB"
msgstr[1] "%(size).0f kB"
msgstr[2] "%(size).0f kB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:436 ../UpdateManager/Core/utils.py:436
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:206 ../DistUpgrade/distro.py:436
#, python-format
msgid "Server for %s"
msgstr "Server pro zemi „%s“"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:224 ../DistUpgrade/distro.py:230
#: ../DistUpgrade/distro.py:246
msgid "Main server"
msgstr "Hlavní server"

#: ../DistUpgrade/distro.py:250
msgid "Custom servers"
msgstr "Uživatelem vybrané servery"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:142
msgid "Could not calculate sources.list entry"
msgstr "Nebylo možné vypočítat záznam v sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:251
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Nelze najít žádné soubory balíčků, pravděpodobně toto není disk Ubuntu nebo "
"obsahuje nesprávnou architekturu?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:294
msgid "Failed to add the CD"
msgstr "Selhalo přidávání CD"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:295
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Vyskytla se chyba při přidávání CD, přechod na vyšší verzi bude přerušen. "
"Prosím nahlaste tuto situaci jako chybu (pokud používáte správné CD s "
"Ubuntu).\n"
"\n"
"Chybová zpráva byla:\n"
"„%s“"

#: ../DistUpgrade/DistUpgradeCache.py:151
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Odstranit balíček ve špatném stavu"
msgstr[1] "Odstranit balíčky ve špatném stavu"
msgstr[2] "Odstranit balíčky ve špatném stavu"

#: ../DistUpgrade/DistUpgradeCache.py:154
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Balíček „%s“ je v nekonzistentním stavu a vyžaduje přeinstalování, ale nebyl "
"pro něj nalezen žádný archiv. Chcete nyní pokračovat odstraněním tohoto "
"balíčku?"
msgstr[1] ""
"Balíčky „%s“ jsou v nekonzistentním stavu a vyžadují přeinstalování, ale "
"nebyl pro ně nalezen žádný archiv. Chcete nyní pokračovat odstraněním těchto "
"balíčků?"
msgstr[2] ""
"Balíčky „%s“ jsou v nekonzistentním stavu a vyžadují přeinstalování, ale "
"nebyl pro ně nalezen žádný archiv. Chcete nyní pokračovat odstraněním těchto "
"balíčků?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:255
msgid "The server may be overloaded"
msgstr "Server je možná přetížený"

#: ../DistUpgrade/DistUpgradeCache.py:368
msgid "Broken packages"
msgstr "Poškozené balíčky"

#: ../DistUpgrade/DistUpgradeCache.py:369
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Váš systém obsahuje poškozené balíčky, které nemohou být tímto programem "
"opraveny. Před pokračováním je prosím opravte použitím programu synaptic "
"nebo apt-get."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:693
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Během plánování povýšení systému nastal neřešitelný problém:\n"
"%s\n"
"\n"
" Možné příčiny:\n"
" * Povyšování na předprodukční verzi Ubuntu\n"
" * Spuštěný systém je předprodukční verze Ubuntu\n"
" * V systému jsou neoficiální softwarové balíčky neposkytované od Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:703
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Toto je pravděpodobně pouze dočasný problém, prosím zkuste to znovu později."

#: ../DistUpgrade/DistUpgradeCache.py:706
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Pokud nic z toho nepomůže, nahlaste prosím tento problém pomocí příkazu "
"'ubuntu-bug update-manager'."

#: ../DistUpgrade/DistUpgradeCache.py:711
#: ../UpdateManager/UpdateManager.py:1031
msgid "Could not calculate the upgrade"
msgstr "Nelze vypočítat přechod na vyšší verzi"

#: ../DistUpgrade/DistUpgradeCache.py:762
msgid "Error authenticating some packages"
msgstr "Chyba při ověřování některých balíčků"

#: ../DistUpgrade/DistUpgradeCache.py:763
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Nebylo možné ověřit pravost některých balíčků. Důvodem může být přechodný "
"problém v síti. Zkuste to prosím později. Níže je uveden seznam postižených "
"balíčků."

#: ../DistUpgrade/DistUpgradeCache.py:783
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Balíček „%s“ je označen k odstranění, ale přitom je uveden v seznamu "
"neodstranitelných."

#: ../DistUpgrade/DistUpgradeCache.py:787
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Stěžejní balíček „%s“ je označen k odstranění."

#: ../DistUpgrade/DistUpgradeCache.py:796
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Pokus o instalaci zakázané verze „%s“"

#: ../DistUpgrade/DistUpgradeCache.py:914
#, python-format
msgid "Can't install '%s'"
msgstr "Nelze nainstalovat „%s“"

#: ../DistUpgrade/DistUpgradeCache.py:915
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug update-manager' in a terminal."
msgstr ""
"Nebylo možné instalovat požadovaný balíček. Nahlaste prosím tuto chybu "
"pomocí příkazu 'ubuntu-bug update-manager' v terminálu."

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:926
msgid "Can't guess meta-package"
msgstr "Nelze odhadnout meta-balíček"

#: ../DistUpgrade/DistUpgradeCache.py:927
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Váš systém neobsahuje žádný z balíčků ubuntu-desktop, kubuntu-desktop, "
"xubuntu-desktop nebo edubuntu-desktop a nebylo možné zjistit, kterou verzi "
"Ubuntu používáte.\n"
" Před tím, než budete pokračovat, si prosím nejprve nainstalujte jeden z "
"uvedených balíčků pomocí programu synapticu nebo apt-get."

#: ../DistUpgrade/DistUpgradeController.py:114
msgid "Reading cache"
msgstr "Probíhá čtení mezipaměti"

#: ../DistUpgrade/DistUpgradeController.py:223
msgid "Unable to get exclusive lock"
msgstr "Nelze získat exkluzivní přístup"

#: ../DistUpgrade/DistUpgradeController.py:224
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Tohle obvykle znamená, že jiný správce balíčků (jako apt-get nebo aptitude) "
"již běží. Nejdříve jej prosím ukončete."

#: ../DistUpgrade/DistUpgradeController.py:257
msgid "Upgrading over remote connection not supported"
msgstr "Aktualizace přes vzdálené připojení není podporována"

#: ../DistUpgrade/DistUpgradeController.py:258
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Spouštíte povýšení systému přes vzdálené připojení ssh s nadstavbou, která "
"toto nepodporuje. Zkuste prosím povýšení v textovém režimu pomocí „do-"
"release-upgrade“.\n"
"\n"
"Povýšení systému se nyní přeruší. Zkuste to prosím bez ssh."

#: ../DistUpgrade/DistUpgradeController.py:272
msgid "Continue running under SSH?"
msgstr "Pokračovat s během pod SSH?"

#: ../DistUpgrade/DistUpgradeController.py:273
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Zdá se, že toto sezení běží pod ssh. Aktuálně není doporučeno provádět "
"povýšení systému přes ssh, neboť v případě havárie je komplikované provést "
"obnovu.\n"
"\n"
"Pokud budete pokračovat, dodatečný ssh démon bude spuštěn na portu „%s“.\n"
"Chcete pokračovat?"

#: ../DistUpgrade/DistUpgradeController.py:287
msgid "Starting additional sshd"
msgstr "Spouští se dodatečné sshd"

#: ../DistUpgrade/DistUpgradeController.py:288
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Aby byla obnova případné havárie lehčí, bude na portu „%s“ spuštěn dodatečný "
"sshd. Pokud nastane nějaká chyba s běžícím ssh, můžete se stále připojit na "
"ten dodatečný.\n"

#: ../DistUpgrade/DistUpgradeController.py:296
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Pokud máte spuštěný firewall, budete možná muset dočasně tento port otevřít. "
"Jelikož je to potenciálně nebezpečné, neprovádí se to automaticky. Tento "
"port můžete otevřít např. takto:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:368
#: ../DistUpgrade/DistUpgradeController.py:413
msgid "Can not upgrade"
msgstr "Nelze přejít na vyšší verzi"

#: ../DistUpgrade/DistUpgradeController.py:369
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Povýšení systému z „%s“ na „%s“ není tímto nástrojem podporované."

#: ../DistUpgrade/DistUpgradeController.py:378
msgid "Sandbox setup failed"
msgstr "Selhalo nastavení zabezpečeného prostředí"

#: ../DistUpgrade/DistUpgradeController.py:379
msgid "It was not possible to create the sandbox environment."
msgstr "Nebylo možné vytvořit zabezpečené prostředí."

#: ../DistUpgrade/DistUpgradeController.py:385
msgid "Sandbox mode"
msgstr "Režim zabezpečeného prostředí"

#: ../DistUpgrade/DistUpgradeController.py:386
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""
"Toto povýšení běží v testovacím režimu (sandbox). Všechny změny jsou zapsány "
"do '%s' a budou ztraceny při následujícím restartu.\n"
"Žádné změny zapsané do systémového adresáře do následujícího restartu nejsou "
"trvalé!"

#: ../DistUpgrade/DistUpgradeController.py:414
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Vaše instalace pythonu je poškozena. Prosím opravte symbolický odkaz „/usr/"
"bin/python“."

#: ../DistUpgrade/DistUpgradeController.py:440
msgid "Package 'debsig-verify' is installed"
msgstr "Balíček „debsig-verify“ je nainstalován"

#: ../DistUpgrade/DistUpgradeController.py:441
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"Aktualizace nemůže pokračovat, dokud je tento balíček nainstalován.\n"
"Nejprve prosím odstraňte balíček pomocí programu Synaptic nebo „apt-get "
"remove debsig-verify“ a poté spusťte aktualizaci znovu."

#: ../DistUpgrade/DistUpgradeController.py:453
#, python-format
msgid "Can not write to '%s'"
msgstr "Nelze zapisovat do '%s'"

#: ../DistUpgrade/DistUpgradeController.py:454
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Ve vašem systému není možné zapisovat do systémového adresáře '%s'. Povýšení "
"nemůže pokračovat.\n"
"Prosím, ujistěte se, že systémový adresář je zapisovatelný."

#: ../DistUpgrade/DistUpgradeController.py:465
msgid "Include latest updates from the Internet?"
msgstr "Včetně nejnovějších aktualizací z internetu?"

#: ../DistUpgrade/DistUpgradeController.py:466
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Aktualizační systém může automaticky stáhnout nejnovější aktualizace a "
"nainstalovat je během povyšování. Pokud máte síťové připojení, pak je toto "
"vřele doporučeno.\n"
"\n"
"Povyšování bude trvat déle, ale po skončení budete mít zcela aktuální "
"systém. Nemusíte to provádět, ale brzo po povýšení systému byste měli "
"nainstalovat nejnovější aktualizace.\n"
"Pokud zde odpovíte „ne“, tak nebude síť vůbec použita."

#: ../DistUpgrade/DistUpgradeController.py:686
#, python-format
msgid "disabled on upgrade to %s"
msgstr "zakázáno při povýšení na %s"

#: ../DistUpgrade/DistUpgradeController.py:713
msgid "No valid mirror found"
msgstr "Nebyl nalezen platný zrcadlový server"

#: ../DistUpgrade/DistUpgradeController.py:714
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Během prohledávání informací o vašich zdrojích nebylo nalezeno žádné zrcadlo "
"pro povýšení. Toto může nastat, pokud provozujete lokální zrcadlo nebo pokud "
"jsou informace na zrcadlu zastaralé.\n"
"\n"
"Chcete přesto přepsat váš soubor „sources.list“? Pokud zvolíte „Ano“, "
"aktualizuje to všech „%s“ na „%s“ položek.\n"
"Pokud zvolíte „Ne“, povýšení se zruší."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:734
msgid "Generate default sources?"
msgstr "Vygenerovat výchozí zdroje?"

#: ../DistUpgrade/DistUpgradeController.py:735
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Po prohledání vašeho souboru „sources.list“ nebyla nalezena žádná platná "
"položka pro „%s“.\n"
"\n"
"Mají být přidány výchozí položky pro „%s“? Pokud zvolíte „Ne“, přechod na "
"vyšší verzi bude zrušen."

#: ../DistUpgrade/DistUpgradeController.py:770
msgid "Repository information invalid"
msgstr "Neplatná informace o repozitáři"

#: ../DistUpgrade/DistUpgradeController.py:771
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"Povýšení informace o repozitáři vyústilo v neplatný soubor, zahajuje se "
"proces hlášení chyby."

#: ../DistUpgrade/DistUpgradeController.py:778
msgid "Third party sources disabled"
msgstr "Zdroje třetích stran zakázány"

#: ../DistUpgrade/DistUpgradeController.py:779
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Některé položky třetích stran ve vašem souboru sources.list byly zakázány. "
"Můžete je znovu povolit po aktualizaci nástrojem „software-properties“ nebo "
"svým správcem balíčků."

#: ../DistUpgrade/DistUpgradeController.py:819
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Balíček je v nekonzistentním stavu"
msgstr[1] "Balíčky jsou v nekonzistentním stavu"
msgstr[2] "Balíčky jsou v nekonzistentním stavu"

#: ../DistUpgrade/DistUpgradeController.py:822
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Balíček „%s“ je v nekonzistentním stavu a vyžaduje přeinstalování, ale nebyl "
"pro něj nalezen žádný archiv. Přeinstalujte prosím balíček ručně nebo jej "
"odeberte ze systému."
msgstr[1] ""
"Balíčky „%s“ jsou v nekonzistentním stavu a vyžadují přeinstalování, ale "
"nebyl pro ně nalezen žádný archiv. Přeinstalujte prosím balíčky ručně nebo "
"je odeberte ze systému."
msgstr[2] ""
"Balíčky „%s“ jsou v nekonzistentním stavu a vyžadují přeinstalování, ale "
"nebyl pro ně nalezen žádný archiv. Přeinstalujte prosím balíčky ručně nebo "
"je odeberte ze systému."

#: ../DistUpgrade/DistUpgradeController.py:870
msgid "Error during update"
msgstr "Chyba během aktualizace"

#: ../DistUpgrade/DistUpgradeController.py:871
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Během aktualizace se vyskytl problém. Obvykle to bývá způsobeno chybou "
"síťového připojení. Zkontrolujte prosím své připojení k síti a zkuste to "
"znovu."

#. print("on_button_install_clicked")
#: ../DistUpgrade/DistUpgradeController.py:880
#: ../UpdateManager/UpdateManager.py:757
msgid "Not enough free disk space"
msgstr "Nedostatek volného místa na disku"

#: ../DistUpgrade/DistUpgradeController.py:881
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"Přechod na novější vydání byl přerušen. Přechod na novější vydání vyžaduje "
"celkem %s volného místa na disku „%s“. Uvolněte prosím přinejmenším dalších "
"%s místa na disku „%s“. Vyprázdněte koš a odstraňte dočasné balíčky z "
"dřívějších instalací příkazem „sudo apt-get clean“."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:910
#: ../DistUpgrade/DistUpgradeController.py:1692
msgid "Calculating the changes"
msgstr "Propočítávají se změny"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:942
msgid "Do you want to start the upgrade?"
msgstr "Chcete spustit aktualizaci?"

#: ../DistUpgrade/DistUpgradeController.py:1008
msgid "Upgrade canceled"
msgstr "Přechod na vyšší verzi zrušen"

#: ../DistUpgrade/DistUpgradeController.py:1009
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Přechod na vyšší verzi bude nyní zrušen a dojde k obnovení systému do "
"původního stavu. Pokračovat v přechodu na vyšší verzi můžete později."

#: ../DistUpgrade/DistUpgradeController.py:1015
#: ../DistUpgrade/DistUpgradeController.py:1149
msgid "Could not download the upgrades"
msgstr "Nepodařilo se stáhnout novější vydání"

#: ../DistUpgrade/DistUpgradeController.py:1016
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Upgrade byl přerušen. Prosím zkontrolujte vaše internetové připojení nebo "
"instalační médium a zkuste to znovu. Všechny prozatím stažené soubory jsou "
"podrženy."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1100
#: ../DistUpgrade/DistUpgradeController.py:1137
#: ../DistUpgrade/DistUpgradeController.py:1242
msgid "Error during commit"
msgstr "Chyba při potvrzování"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1102
#: ../DistUpgrade/DistUpgradeController.py:1139
#: ../DistUpgrade/DistUpgradeController.py:1281
msgid "Restoring original system state"
msgstr "Obnovuje se původní stav systému"

#: ../DistUpgrade/DistUpgradeController.py:1103
#: ../DistUpgrade/DistUpgradeController.py:1118
#: ../DistUpgrade/DistUpgradeController.py:1140
msgid "Could not install the upgrades"
msgstr "Nepodařilo se nainstalovat aktualizace"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1108
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Přechod na novější vydání byl přerušen. Váš systém by mohl být v "
"nepoužitelném stavu. Nyní bude spuštěna obnova (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1113
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/update-manager/+filebug and attach the files in /var/log/dist-"
"upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Prosím nahlašte tuto chybu ve vašem prohlížeči na adrese http://bugs."
"launchpad.net/ubuntu/+source/update-manager/+filebug a připojte soubory z "
"adresáře  /var/log/dist-upgrade/.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1150
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Přechod na novější vydání byl přerušen. Zkontrolujte prosím své internetové "
"připojení nebo instalační média a zkuste to znovu. "

#: ../DistUpgrade/DistUpgradeController.py:1230
msgid "Remove obsolete packages?"
msgstr "Odstranit zastaralé balíčky?"

#: ../DistUpgrade/DistUpgradeController.py:1231
#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Ponechat"

#: ../DistUpgrade/DistUpgradeController.py:1231
msgid "_Remove"
msgstr "Odst_ranit"

#: ../DistUpgrade/DistUpgradeController.py:1243
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Během úklidu se vyskytl problém. Pro více informací si prosím přečtěte níže "
"uvedenou zprávu. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1319
msgid "Required depends is not installed"
msgstr "Požadované závislosti nejsou nainstalovány"

#: ../DistUpgrade/DistUpgradeController.py:1320
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Požadovaná závislost „%s“ není nainstalována. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1588
#: ../DistUpgrade/DistUpgradeController.py:1653
msgid "Checking package manager"
msgstr "Kontroluje se správce balíčků"

#: ../DistUpgrade/DistUpgradeController.py:1593
msgid "Preparing the upgrade failed"
msgstr "Příprava přechodu na vyšší verzi selhala"

#: ../DistUpgrade/DistUpgradeController.py:1594
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr "Příprava systému na povýšení selhala, proces hlášení chyby zahájen."

#: ../DistUpgrade/DistUpgradeController.py:1608
msgid "Getting upgrade prerequisites failed"
msgstr "Získání potřebných prostředků pro povýšení systému selhalo"

#: ../DistUpgrade/DistUpgradeController.py:1609
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"Systém nemohl obdržet požadavky pro povýšení. Povýšení bude nyní ukončeno a "
"bude obnoven výchozí stav systému.\n"
"\n"
"Navíc byl zahájen proces hlášení chyby."

#: ../DistUpgrade/DistUpgradeController.py:1637
msgid "Updating repository information"
msgstr "Aktualizují se informace o repozitářích"

#: ../DistUpgrade/DistUpgradeController.py:1644
msgid "Failed to add the cdrom"
msgstr "Nepodařilo se přidat cdrom"

#: ../DistUpgrade/DistUpgradeController.py:1645
msgid "Sorry, adding the cdrom was not successful."
msgstr "Promiňte, přidání cdrom nebylo úspěšné."

#: ../DistUpgrade/DistUpgradeController.py:1673
msgid "Invalid package information"
msgstr "Neplatná informace o balíčku"

#: ../DistUpgrade/DistUpgradeController.py:1674
msgid "After updating your package "
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1698
#: ../DistUpgrade/DistUpgradeController.py:1750
msgid "Fetching"
msgstr "Získává se"

#: ../DistUpgrade/DistUpgradeController.py:1704
#: ../DistUpgrade/DistUpgradeController.py:1754
msgid "Upgrading"
msgstr "Probíhá přechod na vyšší verzi"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1709
#: ../DistUpgrade/DistUpgradeController.py:1756
#: ../DistUpgrade/DistUpgradeController.py:1763
#: ../DistUpgrade/DistUpgradeController.py:1774
msgid "Upgrade complete"
msgstr "Povýšení systému hotovo"

#: ../DistUpgrade/DistUpgradeController.py:1710
#: ../DistUpgrade/DistUpgradeController.py:1757
#: ../DistUpgrade/DistUpgradeController.py:1764
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"Přechod na vyšší verzi byl dokončen, ale během povyšování nastaly chyby."

#: ../DistUpgrade/DistUpgradeController.py:1717
msgid "Searching for obsolete software"
msgstr "Vyhledává se zastaralý software"

#: ../DistUpgrade/DistUpgradeController.py:1726
msgid "System upgrade is complete."
msgstr "Přechod na vyšší verzi systému je dokončen."

#: ../DistUpgrade/DistUpgradeController.py:1775
msgid "The partial upgrade was completed."
msgstr "Částečné povýšení systému bylo dokončeno."

#: ../DistUpgrade/DistUpgradeQuirks.py:204
msgid "evms in use"
msgstr "používán evms"

#: ../DistUpgrade/DistUpgradeQuirks.py:205
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Váš systém používá správce svazků „evms“ v /proc/mounts. Jelikož software "
"„evms“ již není nadále podporován, vypněte jej prosím, a poté spusťte "
"aktualizaci znovu."

#: ../DistUpgrade/DistUpgradeQuirks.py:502
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr "Vaše grafická karta nemusí být v Ubuntu 12.04 LTS plně podporována."

#: ../DistUpgrade/DistUpgradeQuirks.py:504
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"Podpora vaší grafické karty Intel v Ubuntu 12.04 LTS je omezena a po "
"přechodu se můžete setkat s problémy. Pro více informací navštivte https://"
"wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Přejete si pokračovat v "
"přechodu na novější vydání?"

#: ../DistUpgrade/DistUpgradeQuirks.py:526
#: ../DistUpgrade/DistUpgradeQuirks.py:554
#: ../DistUpgrade/DistUpgradeQuirks.py:581
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Povýšení může omezit grafické efekty prostředí a snížit výkon ve hrách a "
"jiných graficky náročných aplikacích."

#: ../DistUpgrade/DistUpgradeQuirks.py:530
#: ../DistUpgrade/DistUpgradeQuirks.py:558
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Tento počítač v současné době používá grafický ovladač „nvidia“ od NVIDIA. V "
"Ubuntu 10.04 LTS není k dispozici žádná verze tohoto ovladače, která by "
"pracovala s vaší grafickou kartou.\n"
"\n"
"Přejete si pokračovat?"

#: ../DistUpgrade/DistUpgradeQuirks.py:585
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Tento počítač v současné době používá grafický ovladač „fglrx“ od AMD. V "
"Ubuntu 10.04 LTS není k dispozici žádná verze tohoto ovladače, která by "
"pracovala s vaším hardwarem.\n"
"\n"
"Přejete si pokračovat?"

#: ../DistUpgrade/DistUpgradeQuirks.py:615
msgid "No i686 CPU"
msgstr "Nenalezeno CPU i686"

#: ../DistUpgrade/DistUpgradeQuirks.py:616
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Váš systém používá CPU i586 nebo CPU, které nemá rozšíření „cmov“. Veškeré "
"balíky byly sestaveny s optimalizací vyžadující minimální architekturu i686. "
"S tímto hardwarem není možné povýšit váš systém na nové vydání Ubuntu."

#: ../DistUpgrade/DistUpgradeQuirks.py:652
msgid "No ARMv6 CPU"
msgstr "Nemáte procesor ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:653
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Váš systém používá procesor ARM, který je starší než architektura ARMv6. "
"Všechny balíčky ve vydání Karmic byly sestaveny s optimalizacemi "
"požadujícími minimálně architekturu ARMv6. S tímto hardwarem není možné "
"povýšit váš systém na nové vydání Ubuntu."

#: ../DistUpgrade/DistUpgradeQuirks.py:673
msgid "No init available"
msgstr "Není k dispozici žádný démon init"

#: ../DistUpgrade/DistUpgradeQuirks.py:674
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Zdá se, že váš systém běží ve virtualizovaném prostředí bez démona init, "
"např. Linux-VServer. Ubuntu 10.04 LTS nemůže v takovémto typu prostředí "
"pracovat. Proveďte prosím nejprve požadovanou aktualizaci virtuálního "
"stroje.\n"
"\n"
"Opravdu chcete pokračovat?"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "Sandbox upgrade using aufs"
msgstr "Aktualizace v chráněném prostředí za pomoci aufs"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "Použij danou cestu k vyhledání cdrom s aktualizovatelnými balíčky"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Používat nadstavbu. Aktuálně dostupné :\n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "*DEPRECATED* this option will be ignored"
msgstr "*ZASTARALÉ* tato volba bude ignorována"

#: ../DistUpgrade/DistUpgradeMain.py:79
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "Provést pouze částečné povýšení (bez přepisování sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:82
msgid "Disable GNU screen support"
msgstr "Zakázat podporu GNU screen"

#: ../DistUpgrade/DistUpgradeMain.py:84
msgid "Set datadir"
msgstr "Nastavit datadir"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeViewGtk.py:114
#: ../DistUpgrade/DistUpgradeViewGtk3.py:117
#: ../DistUpgrade/DistUpgradeViewKDE.py:195
#: ../UpdateManager/DistUpgradeFetcherKDE.py:155
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Vložte prosím „%s“ do mechaniky „%s“"

#: ../DistUpgrade/DistUpgradeViewGtk.py:135
#: ../DistUpgrade/DistUpgradeViewGtk3.py:138
#: ../DistUpgrade/DistUpgradeViewKDE.py:209
msgid "Fetching is complete"
msgstr "Stahování je dokončeno"

#: ../DistUpgrade/DistUpgradeViewGtk.py:146
#: ../DistUpgrade/DistUpgradeViewGtk3.py:149
#: ../DistUpgrade/DistUpgradeViewKDE.py:222
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Stahování souboru %li z %li při %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:149
#: ../DistUpgrade/DistUpgradeViewGtk.py:296
#: ../DistUpgrade/DistUpgradeViewGtk3.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:309
#: ../DistUpgrade/DistUpgradeViewKDE.py:223
#: ../DistUpgrade/DistUpgradeViewKDE.py:371
#, python-format
msgid "About %s remaining"
msgstr "Zbývá zhruba %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:152
#: ../DistUpgrade/DistUpgradeViewGtk3.py:155
#: ../DistUpgrade/DistUpgradeViewKDE.py:225
#, python-format
msgid "Fetching file %li of %li"
msgstr "Stahování souboru %li z %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewGtk3.py:186
#: ../DistUpgrade/DistUpgradeViewKDE.py:262
msgid "Applying changes"
msgstr "Provádějí se změny"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewGtk3.py:212
#: ../DistUpgrade/DistUpgradeViewKDE.py:275
msgid "dependency problems - leaving unconfigured"
msgstr "problém se závislostmi - ponecháno nenastavené"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewGtk3.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:277
#, python-format
msgid "Could not install '%s'"
msgstr "Nelze nainstalovat „%s“"

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewGtk3.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:278
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Přechod na vyšší verzi bude pokračovat, ale balíček „%s“ nemusí pracovat "
"správně. Zvažte prosím odeslání hlášení o chybě."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewGtk3.py:235
#: ../DistUpgrade/DistUpgradeViewKDE.py:299
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Nahradit upravený soubor s nastavením\n"
"„%s“?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewGtk3.py:236
#: ../DistUpgrade/DistUpgradeViewKDE.py:300
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Pokud zvolíte nahrazení novější verzí, ztratíte veškeré lokální změny tohoto "
"konfiguračního souboru."

#: ../DistUpgrade/DistUpgradeViewGtk.py:251
#: ../DistUpgrade/DistUpgradeViewGtk3.py:256
#: ../DistUpgrade/DistUpgradeViewKDE.py:323
msgid "The 'diff' command was not found"
msgstr "Příkaz „diff“ nebyl nalezen"

#: ../DistUpgrade/DistUpgradeViewGtk.py:464
#: ../DistUpgrade/DistUpgradeViewGtk3.py:477
#: ../DistUpgrade/DistUpgradeViewText.py:92
msgid "A fatal error occurred"
msgstr "Došlo ke kritické chybě"

#: ../DistUpgrade/DistUpgradeViewGtk.py:465
#: ../DistUpgrade/DistUpgradeViewGtk3.py:478
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Toto prosím nahlaste jako chybu (pokud jste tak již neučinili) a přidejte ke "
"své zprávě soubory /var/log/dist-upgrade/main.log a /var/log/dist-upgrade/"
"apt.log. Přechod na novější vydání byl přerušen.\n"
"Váš původní soubor sources.list byl uložen do /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:482
#: ../DistUpgrade/DistUpgradeViewGtk3.py:495
msgid "Ctrl-c pressed"
msgstr "Stisknuto Ctrl-c"

#: ../DistUpgrade/DistUpgradeViewGtk.py:483
#: ../DistUpgrade/DistUpgradeViewGtk3.py:496
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Toto přeruší operaci a může zanechat systém v rozbitém stavu. Jste si jisti, "
"že to chcete provést?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:631
#: ../DistUpgrade/DistUpgradeViewGtk3.py:629
msgid "To prevent data loss close all open applications and documents."
msgstr "Pro zamezení ztráty dat uzavřete všechny aplikace a dokumenty."

#: ../DistUpgrade/DistUpgradeViewGtk.py:645
#: ../DistUpgrade/DistUpgradeViewGtk3.py:643
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Již není podporováno společností Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:646
#: ../DistUpgrade/DistUpgradeViewGtk3.py:644
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Přejít na nižší verzi (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:647
#: ../DistUpgrade/DistUpgradeViewGtk3.py:645
#, python-format
msgid "Remove (%s)"
msgstr "Odstranit (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:648
#: ../DistUpgrade/DistUpgradeViewGtk3.py:646
#, python-format
msgid "No longer needed (%s)"
msgstr "Již není potřeba (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:649
#: ../DistUpgrade/DistUpgradeViewGtk3.py:647
#, python-format
msgid "Install (%s)"
msgstr "Nainstalovat (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:650
#: ../DistUpgrade/DistUpgradeViewGtk3.py:648
#, python-format
msgid "Upgrade (%s)"
msgstr "Přejít na vyšší verzi (%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:196
#: ../UpdateManager/DistUpgradeFetcherKDE.py:157
msgid "Media Change"
msgstr "Změna média"

#: ../DistUpgrade/DistUpgradeViewKDE.py:335
msgid "Show Difference >>>"
msgstr "Zobrazit rozdíl >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:338
msgid "<<< Hide Difference"
msgstr "<<< Skrýt rozdíl"

#: ../DistUpgrade/DistUpgradeViewKDE.py:554
msgid "Error"
msgstr "Chyba"

#: ../DistUpgrade/DistUpgradeViewKDE.py:568
msgid "&Cancel"
msgstr "&Zrušit"

#: ../DistUpgrade/DistUpgradeViewKDE.py:572
#: ../DistUpgrade/DistUpgradeViewKDE.py:813
msgid "&Close"
msgstr "&Zavřít"

#: ../DistUpgrade/DistUpgradeViewKDE.py:618
msgid "Show Terminal >>>"
msgstr "Zobrazit terminál >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:621
msgid "<<< Hide Terminal"
msgstr "<<< Skrýt terminál"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Information"
msgstr "Informace"

#: ../DistUpgrade/DistUpgradeViewKDE.py:751
#: ../DistUpgrade/DistUpgradeViewKDE.py:796
#: ../DistUpgrade/DistUpgradeViewKDE.py:799 ../DistUpgrade/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Podrobnosti"

#: ../DistUpgrade/DistUpgradeViewKDE.py:777
#, python-format
msgid "No longer supported %s"
msgstr "Již není podporováno %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:779
#, python-format
msgid "Remove %s"
msgstr "Odstranit %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:781
#: ../DistUpgrade/DistUpgradeViewText.py:182
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Odstranit (byl automaticky nainstalován) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:783
#, python-format
msgid "Install %s"
msgstr "Nainstalovat %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#, python-format
msgid "Upgrade %s"
msgstr "Aktualizovat %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:809
#: ../DistUpgrade/DistUpgradeViewText.py:230
msgid "Restart required"
msgstr "Je nutný restart"

#: ../DistUpgrade/DistUpgradeViewKDE.py:809
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr ""
"<b><big>Pro dokončení přechodu na vyšší verzi systému restartujte počítač</"
"big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:812 ../DistUpgrade/DistUpgrade.ui.h:14
#: ../data/gtkbuilder/UpdateManager.ui.h:26
msgid "_Restart Now"
msgstr "_Restartovat nyní"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:830
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Zrušit probíhající povýšení systému?</big></b>\n"
"\n"
"Zrušení povýšení může systém uvést do nepoužitelného stavu. Důrazně "
"doporučujeme v povýšení pokračovat."

#: ../DistUpgrade/DistUpgradeViewKDE.py:834
msgid "Cancel Upgrade?"
msgstr "Zrušit povyšování systému?"

#: ../DistUpgrade/DistUpgradeView.py:61
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li den"
msgstr[1] "%li dny"
msgstr[2] "%li dní"

#: ../DistUpgrade/DistUpgradeView.py:63
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li hodina"
msgstr[1] "%li hodiny"
msgstr[2] "%li hodin"

#: ../DistUpgrade/DistUpgradeView.py:65
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minuta"
msgstr[1] "%li minuty"
msgstr[2] "%li minut"

#: ../DistUpgrade/DistUpgradeView.py:66
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekunda"
msgstr[1] "%li sekundy"
msgstr[2] "%li sekund"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:100
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Stahování zabere přibližně %s s 1Mbit připojením DSL a přibližně %s s 56k "
"modemem."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:155
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Stahování bude s vaším připojením trvat cca %s. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:259 ../DistUpgrade/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "Připravuje se povýšení systému"

#: ../DistUpgrade/DistUpgradeView.py:260
msgid "Getting new software channels"
msgstr "Získávají se nové softwarové kanály"

#: ../DistUpgrade/DistUpgradeView.py:261 ../DistUpgrade/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "Získávají se nové balíčky"

#: ../DistUpgrade/DistUpgradeView.py:262 ../DistUpgrade/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "Instalují se aktualizace"

#: ../DistUpgrade/DistUpgradeView.py:263 ../DistUpgrade/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "Probíhá úklid"

#: ../DistUpgrade/DistUpgradeView.py:348
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d nainstalovaný balík již není podporován společností Canonical. "
"Stále však můžete získat podporu od komunity."
msgstr[1] ""
"%(amount)d nainstalované balíky již nejsou podporovány společností "
"Canonical. Stále však můžete získat podporu od komunity."
msgstr[2] ""
"%(amount)d nainstalovaných balíků již není podporováno společností "
"Canonical. Stále však můžete získat podporu od komunity."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:357
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d balíček bude odstraněn."
msgstr[1] "%d balíčky budou odstraněny."
msgstr[2] "%d balíčků bude odstraněno."

#: ../DistUpgrade/DistUpgradeView.py:362
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d nový balíček bude nainstalován."
msgstr[1] "%d nové balíčky budou nainstalovány."
msgstr[2] "%d nových balíčků bude nainstalováno."

#: ../DistUpgrade/DistUpgradeView.py:368
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d balíček bude nahrazen vyšší verzí."
msgstr[1] "%d balíčky budou nahrazeny vyšší verzí."
msgstr[2] "%d balíčků bude nahrazeno vyšší verzí."

#: ../DistUpgrade/DistUpgradeView.py:373
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Bude staženo celkem %s. "

#: ../DistUpgrade/DistUpgradeView.py:378
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Instalace aktualizací může zabrat až několik hodin. Jakmile bude dokončeno "
"stahování, nelze proces ukončit."

#: ../DistUpgrade/DistUpgradeView.py:382
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Stažení a instalace aktualizací může zabrat až několik hodin. Jakmile bude "
"dokončeno stahování, nelze proces ukončit."

#: ../DistUpgrade/DistUpgradeView.py:387
msgid "Removing the packages can take several hours. "
msgstr "Mazání balíků může trvat několik hodin. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:392 ../UpdateManager/UpdateManager.py:676
msgid "The software on this computer is up to date."
msgstr "Software v tomto počítači je aktuální"

#: ../DistUpgrade/DistUpgradeView.py:393
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Pro váš systém nejsou k dispozici žádné aktualizace. Přechod na vyšší verzi "
"bude nyní zrušen."

#: ../DistUpgrade/DistUpgradeView.py:406
msgid "Reboot required"
msgstr "Je nutné restartovat počítač"

#: ../DistUpgrade/DistUpgradeView.py:407
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Přechod na vyšší verzi byl dokončen a nyní je nutné restartovat počítač. "
"Chcete jej restartovat nyní?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:72
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:72
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "ověřit '%(file)s' proti '%(signature)s' "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:131
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:131
#, python-format
msgid "extracting '%s'"
msgstr "rozbaluje se '%s'"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:151
#: ../DistUpgrade/DistUpgradeFetcherCore.py:152
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:151
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:152
msgid "Could not run the upgrade tool"
msgstr "Nelze spustit nástroj pro aktualizaci"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:152
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:152
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug update-manager'."
msgstr ""
"S největší pravděpodobností se jedná o chybu v nástroji pro přechod. "
"Nahlaste to prosím jako chybu pomocí příkazu 'ubuntu-bug update-manager'."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:227
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:227
msgid "Upgrade tool signature"
msgstr "Podpis aktualizačního nástroje"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:234
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:234
msgid "Upgrade tool"
msgstr "Aktualizační nástroj"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:268
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:268
msgid "Failed to fetch"
msgstr "Stahování selhalo"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "Stažení aktualizace selhalo. Pravděpodobně je problém se sítí. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:273
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:273
msgid "Authentication failed"
msgstr "Selhalo ověření pravosti"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:274
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:274
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Ověření pravosti aktualizace selhalo. Může jít o problém se sítí nebo se "
"serverem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:279
msgid "Failed to extract"
msgstr "Rozbalení selhalo"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:280
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:280
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Rozbalení aktualizace selhalo. Může být problém se sítí nebo se serverem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:285
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:285
msgid "Verification failed"
msgstr "Ověření selhalo"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:286
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:286
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Ověření aktualizace selhalo. Může jít o problém se sítí nebo se serverem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:300
#: ../DistUpgrade/DistUpgradeFetcherCore.py:306
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:300
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:306
msgid "Can not run the upgrade"
msgstr "Nelze spustit povýšení systému"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:301
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:301
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Toto se obvykle stává v pokud je /tmp připojen jako noexec. Prosím připojte /"
"tmp bez noexec a spusťte upgrade znovu."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:307
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:307
#, python-format
msgid "The error message is '%s'."
msgstr "Chybová zpráva zní „%s“."

#: ../DistUpgrade/DistUpgradeViewText.py:93
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Nahlaste prosím toto jako chybu a přidejte ke své zprávě soubory /var/log/"
"dist-upgrade/main.log a /var/log/dist-upgrade/apt.log. Přechod na novější "
"vydání byl přerušen.\n"
"Váš původní soubor sources.list byl uložen do /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:117
msgid "Aborting"
msgstr "Přerušuje se"

#: ../DistUpgrade/DistUpgradeViewText.py:122
msgid "Demoted:\n"
msgstr "Odrazováno:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:129
msgid "To continue please press [ENTER]"
msgstr "Chcete-li pokračovat, stiskněte prosím [ENTER]"

#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:196
#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "Continue [yN] "
msgstr "Pokračovat [aN] "

#: ../DistUpgrade/DistUpgradeViewText.py:157
#: ../DistUpgrade/DistUpgradeViewText.py:196
msgid "Details [d]"
msgstr "Podrobnosti [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:162
#: ../DistUpgrade/DistUpgradeViewText.py:206
msgid "y"
msgstr "a"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:165
#: ../DistUpgrade/DistUpgradeViewText.py:213
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:168
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:173
#, python-format
msgid "No longer supported: %s\n"
msgstr "Již není podporováno: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
#, python-format
msgid "Remove: %s\n"
msgstr "Odstranit: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:188
#, python-format
msgid "Install: %s\n"
msgstr "Nainstalovat: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:193
#, python-format
msgid "Upgrade: %s\n"
msgstr "Aktualizovat: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:210
msgid "Continue [Yn] "
msgstr "Pokračovat [An] "

#: ../DistUpgrade/DistUpgradeViewText.py:231
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Pro dokončení povýšení systému je nutné jej restartovat.\n"
"Pokud zvolíte „a“, tak bude systém restartován."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Zrušit přechod na vyšší verzi"

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "Pok_račovat v přechodu na vyšší verzi"

#: ../DistUpgrade/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Zrušit probíhající aktualizaci?</big></b>\n"
"\n"
"Zrušení aktualizace může systém uvést do nepoužitelného stavu. <i>Důrazně</"
"i> se doporučuje v aktualizaci pokračovat."

#: ../DistUpgrade/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Začít přechod na vyšší verzi"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "Nah_radit"

#: ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Rozdíl mezi soubory"

#: ../DistUpgrade/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Oznámit chybu"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Pokračovat"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Spustit přechod na vyšší verzi systému?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Pro dokončení povýšení systému restartujte počítač</big></b>\n"
"\n"
"Prosím, uložte si před pokračováním svou práci."

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Přechod na vyšší verzi distribuce"

#: ../DistUpgrade/DistUpgrade.ui.h:19
#, fuzzy
msgid "<b><big>Upgrading Ubuntu to version 12.10</big></b>"
msgstr "<b><big>Probíhá povýšení Ubuntu na verzi 11.10</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "Nastavují se nové softwarové kanály"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "Restartuje se počítač"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "Terminál"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:64
msgid "Please wait, this can take some time."
msgstr "Čekejte prosím, může to chvíli trvat."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:66
msgid "Update is complete"
msgstr "Aktualizace je dokončena"

#: ../UpdateManager/DistUpgradeFetcher.py:114
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Could not find the release notes"
msgstr "Nebyly nalezeny poznámky k vydání"

#: ../UpdateManager/DistUpgradeFetcher.py:115
#: ../UpdateManager/DistUpgradeFetcherKDE.py:110
msgid "The server may be overloaded. "
msgstr "Server může být přetížen. "

#: ../UpdateManager/DistUpgradeFetcher.py:125
#: ../UpdateManager/DistUpgradeFetcherKDE.py:114
msgid "Could not download the release notes"
msgstr "Nelze stáhnout poznámky k vydání"

#: ../UpdateManager/DistUpgradeFetcher.py:126
#: ../UpdateManager/DistUpgradeFetcherKDE.py:115
msgid "Please check your internet connection."
msgstr "Zkontrolujte prosím své připojení k internetu."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:68
#: ../UpdateManager/DistUpgradeFetcherKDE.py:91
msgid "Upgrade"
msgstr "Přejít na vyšší verzi"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:95
#: ../data/gtkbuilder/UpdateManager.ui.h:20
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Release Notes"
msgstr "Poznámky k vydání"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:134
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Downloading additional package files..."
msgstr "Stahují se dodatečné balíčky…"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Soubor %s z %s při %s B/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
#, python-format
msgid "File %s of %s"
msgstr "Soubor %s z %s"

#: ../UpdateManager/ChangelogViewer.py:75
msgid "Open Link in Browser"
msgstr "Otevřít odkaz v prohlížeči"

#: ../UpdateManager/ChangelogViewer.py:78
msgid "Copy Link to Clipboard"
msgstr "Kopírovat odkaz do schránky"

#: ../UpdateManager/GtkProgress.py:162
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Stahuje se %(current)li. souboru z %(total)li rychlostí %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:167
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Stahuje se %(current)li. souboru z %(total)li"

#: ../UpdateManager/UpdateManager.py:106 ../do-release-upgrade:100
msgid "Your Ubuntu release is not supported anymore."
msgstr "Vaše vydání Ubuntu již není podporováno."

#: ../UpdateManager/UpdateManager.py:107
msgid ""
"You will not get any further security fixes or critical updates. Please "
"upgrade to a later version of Ubuntu."
msgstr ""
"Nebudete dostávat žádné kritické aktualizace ani bezpečnostní záplaty. "
"Prosím přejděte na novější verzi Ubuntu."

#: ../UpdateManager/UpdateManager.py:115
msgid "Upgrade information"
msgstr "Informace o přechodu na vyšší verzi"

#: ../UpdateManager/UpdateManager.py:233
#: ../UpdateManagerText/UpdateManagerText.py:35
msgid "Install"
msgstr "Nainstalovat"

#: ../UpdateManager/UpdateManager.py:235
msgid "Name"
msgstr "Název"

#. upload_archive = version_match.group(2).strip()
#: ../UpdateManager/UpdateManager.py:395
#, python-format
msgid "Version %s: \n"
msgstr "Verze %s: \n"

#: ../UpdateManager/UpdateManager.py:453
msgid ""
"No network connection detected, you can not download changelog information."
msgstr ""
"Nebylo rozpoznáno žádné připojení k síti, tudíž nemůžete stáhnout seznam "
"změn."

#: ../UpdateManager/UpdateManager.py:463
msgid "Downloading list of changes..."
msgstr "Stahuje se seznam změn…"

#: ../UpdateManager/UpdateManager.py:507
msgid "_Deselect All"
msgstr "O_dznačit vše"

#: ../UpdateManager/UpdateManager.py:513
msgid "Select _All"
msgstr "V_ybrat vše"

#: ../UpdateManager/UpdateManager.py:572
#, python-format
msgid "%s will be downloaded."
msgstr "Bude staženo %s."

#: ../UpdateManager/UpdateManager.py:584
#, fuzzy
msgid "The update has already been downloaded."
msgid_plural "The updates have already been downloaded."
msgstr[0] "Aktualizace byla již stažena, ovšem není nainstalována."
msgstr[1] "Aktualizace byly již staženy, avšak nejsou nainstalovány."
msgstr[2] "Aktualizace byly již staženy, avšak nejsou nainstalovány."

#: ../UpdateManager/UpdateManager.py:589
msgid "There are no updates to install."
msgstr "Nejsou žádné aktualizace k instalaci"

#: ../UpdateManager/UpdateManager.py:598
msgid "Unknown download size."
msgstr "Neznámá velikost stahování."

#: ../UpdateManager/UpdateManager.py:624
msgid ""
"It is unknown when the package information was updated last. Please click "
"the 'Check' button to update the information."
msgstr ""
"Není známo kdy byly informace o balíčcích naposledy obnoveny. Klikněte "
"prosím na tlačítko 'Zkontrolovat' pro aktualizaci informací."

#: ../UpdateManager/UpdateManager.py:630
#, python-format
msgid ""
"The package information was last updated %(days_ago)s days ago.\n"
"Press the 'Check' button below to check for new software updates."
msgstr ""
"Informace o balících byly naposledy aktualizovány před %(days_ago)s dny.\n"
"Klikněte níže na tlačítko „Zkontrolovat“, abyste zkontrolovali nové "
"aktualizace softwaru."

#: ../UpdateManager/UpdateManager.py:635
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural "The package information was last updated %(days_ago)s days ago."
msgstr[0] ""
"Informace o balíčcích byly naposledy aktualizovány před %(days_ago)s dnem."
msgstr[1] ""
"Informace o balíčcích byly naposledy aktualizovány před %(days_ago)s dny."
msgstr[2] ""
"Informace o balíčcích byly naposledy aktualizovány před %(days_ago)s dny."

#: ../UpdateManager/UpdateManager.py:639
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] ""
"Informace o balíčcích byly naposledy aktualizovány před %(hours_ago)s "
"hodinou."
msgstr[1] ""
"Informace o balíčcích byly naposledy aktualizovány před %(hours_ago)s "
"hodinami."
msgstr[2] ""
"Informace o balíčcích byly naposledy aktualizovány před %(hours_ago)s "
"hodinami."

#. TRANSLATORS: only in plural form, as %s minutes ago is one of 15, 30, 45 minutes ago
#: ../UpdateManager/UpdateManager.py:644 ../UpdateManager/UpdateManager.py:646
#: ../UpdateManager/UpdateManager.py:648
#, python-format
msgid "The package information was last updated about %s minutes ago."
msgstr "Informace o balících byly naposledy aktualizovány před %s minutami."

#: ../UpdateManager/UpdateManager.py:650
msgid "The package information was just updated."
msgstr "Informace o balících byly právě aktualizovány."

#: ../UpdateManager/UpdateManager.py:689
msgid "Software updates may be available for your computer."
msgstr "Mohou být dostupné aktualizace softwaru pro váš počítač."

#: ../UpdateManager/UpdateManager.py:697
#, python-format
msgid ""
"Updated software has been issued since %s was released. Do you want to "
"install it now?"
msgstr ""

#: ../UpdateManager/UpdateManager.py:700
msgid ""
"Updated software is available for this computer. Do you want to install it "
"now?"
msgstr ""

#: ../UpdateManager/UpdateManager.py:758
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"Tato aktualizace potřebuje celkem %s volného místa na disku „%s“. Uvolněte "
"prosím alespoň %s dodatečného místa na „%s“. Vyprázdněte koš a smažte "
"dočasné balíčky dřívějších instalací použitím „sudo apt-get clean“."

#: ../UpdateManager/UpdateManager.py:783
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr ""
"K dokončení instalace aktualizací je třeba restartovat počítač. Před "
"pokračováním si prosím uložte svou práci."

#: ../UpdateManager/UpdateManager.py:847
msgid "Reading package information"
msgstr "Čtou se informace o balíčcích"

#: ../UpdateManager/UpdateManager.py:862
msgid "Connecting..."
msgstr "Připojování..."

#: ../UpdateManager/UpdateManager.py:879
msgid "You may not be able to check for updates or download new updates."
msgstr ""
"Možná nebudete moci zkontrolovat aktualizace nebo stahovat nové aktualizace."

#: ../UpdateManager/UpdateManager.py:1002
msgid "Could not initialize the package information"
msgstr "Nelze zjistit informace o balíčku"

#: ../UpdateManager/UpdateManager.py:1003
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Během sbírání informací o balíčku nastal neřešitelný problém.\n"
"\n"
"Nahlaste prosím tento problém vůči balíčku „update-manager“ a přiložte "
"následující chybovou hlášku:\n"

#: ../UpdateManager/UpdateManager.py:1032
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Během propočítávání aktualizace nastal neřešitelný problém.\n"
"\n"
"Nahlaste prosím tuto chybu vůči balíčku „update-manager“ a přiložte "
"následující chybovou zprávu:"

#: ../UpdateManager/UpdateManager.py:1056
msgid " (New install)"
msgstr " (Nová instalace)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:1063
#, python-format
msgid "(Size: %s)"
msgstr "(Velikost: %s)"

#: ../UpdateManager/UpdateManager.py:1067
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Z verze %(old_version)s na verzi %(new_version)s"

#: ../UpdateManager/UpdateManager.py:1071
#, python-format
msgid "Version %s"
msgstr "Verze %s"

#: ../UpdateManager/UpdateManager.py:1104 ../do-release-upgrade:112
msgid "Release upgrade not possible right now"
msgstr "Přechod na novější vydání není v tuto chvíli možný"

#: ../UpdateManager/UpdateManager.py:1105 ../do-release-upgrade:113
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Povýšení na novější vydání nemůže být v současné době provedeno. Zkuste to "
"prosím později znovu. Server oznámil: „%s“"

#: ../UpdateManager/UpdateManager.py:1107 ../check-new-release-gtk:126
msgid "Downloading the release upgrade tool"
msgstr "Stahuje se nástroj pro přechod na novější vydání"

#: ../UpdateManager/UpdateManager.py:1114
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Je k dispozici nové vydání Ubuntu „%s“</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1153
msgid "Software index is broken"
msgstr "Seznam softwaru je poškozen"

#: ../UpdateManager/UpdateManager.py:1154
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Není možné instalovat ani odebírat programy. Prosím použijte správce balíčků "
"„Synaptic“ nebo nejdříve spusťte v terminálu „sudo apt-get install -f“ pro "
"nápravu tohoto problému."

#: ../UpdateManager/UnitySupport.py:57
msgid "Check for Updates"
msgstr "Zkontrolovat aktualizace"

#: ../UpdateManager/UnitySupport.py:66
msgid "Install All Available Updates"
msgstr "Nainstalovat všechny dostupné aktualizace"

#: ../UpdateManagerText/UpdateManagerText.py:34
msgid "Cancel"
msgstr "Zrušit"

#: ../UpdateManagerText/UpdateManagerText.py:37
msgid "Changelog"
msgstr "Seznam změn"

#: ../UpdateManagerText/UpdateManagerText.py:40
msgid "Updates"
msgstr "Aktualizace"

#: ../UpdateManagerText/UpdateManagerText.py:53
msgid "Building Updates List"
msgstr "Vytváří se seznam aktualizací"

#: ../UpdateManagerText/UpdateManagerText.py:56
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Normální aktualizace nemohla být připravena, spusťte prosím: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"To může být způsobeno:\n"
" * Předchozí aktualizace nebyla dokončena\n"
" * Problémy s nainstalovaným softwarem\n"
" * Neoficiální softwarové balíčky neposkytované od Ubuntu\n"
" * Normální změny v předprodukční verzi Ubuntu"

#: ../UpdateManagerText/UpdateManagerText.py:125
msgid "Downloading changelog"
msgstr "Stahuje se seznam změn"

#: ../UpdateManager/Core/MyCache.py:147
#, python-format
msgid "Other updates (%s)"
msgstr "Ostatní aktualizace (%s)"

#: ../UpdateManager/Core/MyCache.py:325
msgid "This update does not come from a source that supports changelogs."
msgstr "Tato aktualizace pochází ze zdroje, který nepodporuje zobrazení změn."

#: ../UpdateManager/Core/MyCache.py:331 ../UpdateManager/Core/MyCache.py:359
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Stažení seznamu změn selhalo. \n"
"Zkontrolujte prosím své internetové připojení."

#: ../UpdateManager/Core/MyCache.py:338
#, python-format
msgid ""
"Changes for the versions:\n"
"Installed version: %s\n"
"Available version: %s\n"
"\n"
msgstr ""
"Změny verzí:\n"
"Nainstalovaná verze: %s\n"
"Dostupná verze: %s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:348
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Seznam změn neobsahuje žádné odpovídající změny.\n"
"\n"
"Použijte prosím http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"než budou změny dostupné nebo opakujte akci později."

#: ../UpdateManager/Core/MyCache.py:353
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Seznam změn ještě není dostupný.\n"
"\n"
"Použijte prosím http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"dokud nebudou změny dostupné, nebo to zkuste znovu později."

#: ../UpdateManager/Core/UpdateList.py:51
msgid "Failed to detect distribution"
msgstr "Detekce distribuce selhala"

#: ../UpdateManager/Core/UpdateList.py:52
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "Během zjišťování, jaký systém používáte, došlo k chybě „%s“."

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Important security updates"
msgstr "Důležité bezpečnostní aktualizace"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Recommended updates"
msgstr "Doporučené aktualizace"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Proposed updates"
msgstr "Navržené aktualizace"

#: ../UpdateManager/Core/UpdateList.py:66
msgid "Backports"
msgstr "Aktualizace přenesené z novějších verzí distribuce"

#: ../UpdateManager/Core/UpdateList.py:67
msgid "Distribution updates"
msgstr "Aktualizace distribuce"

#: ../UpdateManager/Core/UpdateList.py:72
msgid "Other updates"
msgstr "Ostatní aktualizace"

#: ../data/gtkbuilder/UpdateManager.ui.h:1
#, fuzzy
msgid "<big><b>Starting Software Updater</b></big>"
msgstr "<big><b>Spouští se Správce aktualizací</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:2
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Aktualizace softwaru opravují chyby, odstraňují slabá místa v zabezpečení a "
"přinášejí nové funkce."

#: ../data/gtkbuilder/UpdateManager.ui.h:3
msgid "_Partial Upgrade"
msgstr "Čá_stečné povýšení"

#: ../data/gtkbuilder/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Není možné nainstalovat všechny aktualizace</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:5
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Pro instalaci co největšího množství aktualizací spusťte částečné povýšení "
"systému. \n"
"\n"
"Toto může být způsobeno:\n"
" * Předchozím nedokončeným povýšením systému\n"
" * Problémy s některým nainstalovaným softwarem\n"
" * Neoficiálními softwarovými balíčky neposkytovanými od Ubuntu\n"
" * Normálními změnami vývojové verze Ubuntu"

#: ../data/gtkbuilder/UpdateManager.ui.h:12
msgid "Chec_k"
msgstr "Z_kontrolovat"

#: ../data/gtkbuilder/UpdateManager.ui.h:13
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>Aktualizace musíte kontrolovat ručně</big></b>\n"
"\n"
"Váš systém nezjišťuje nové aktualizace automaticky. Toto nastavení můžete "
"změnit v nástroji <i>Zdroje softwaru</i> na kartě <i>Aktualizace</i>."

#: ../data/gtkbuilder/UpdateManager.ui.h:16
msgid "_Hide this information in the future"
msgstr "Příště tyto informace _nezobrazovat"

#: ../data/gtkbuilder/UpdateManager.ui.h:17
msgid "Co_ntinue"
msgstr "Pokračo_vat"

#: ../data/gtkbuilder/UpdateManager.ui.h:18
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Provoz na baterii</b></big>"

#: ../data/gtkbuilder/UpdateManager.ui.h:19
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr "Váš systém běží na baterii. Opravdu chcete pokračovat ?"

#: ../data/gtkbuilder/UpdateManager.ui.h:21
msgid "_Upgrade"
msgstr "_Přejít na vyšší verzi"

#: ../data/gtkbuilder/UpdateManager.ui.h:22
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:8
msgid "Show progress of individual files"
msgstr "Zobrazit průběh jednotlivých souborů"

#: ../data/gtkbuilder/UpdateManager.ui.h:23
#: ../data/update-manager.desktop.in.h:1
#, fuzzy
msgid "Software Updater"
msgstr "Aktualizace softwaru"

#: ../data/gtkbuilder/UpdateManager.ui.h:24
#, fuzzy
msgid "Starting Software Updater"
msgstr "Aktualizace softwaru"

#: ../data/gtkbuilder/UpdateManager.ui.h:25
msgid "U_pgrade"
msgstr "_Přejít na vyšší verzi"

#: ../data/gtkbuilder/UpdateManager.ui.h:27
msgid "updates"
msgstr "aktualizace"

#: ../data/gtkbuilder/UpdateManager.ui.h:28
msgid "Changes"
msgstr "Změny"

#: ../data/gtkbuilder/UpdateManager.ui.h:29
msgid "Description"
msgstr "Popis"

#: ../data/gtkbuilder/UpdateManager.ui.h:30
#, fuzzy
msgid "Details of updates"
msgstr "Popis aktualizace"

#: ../data/gtkbuilder/UpdateManager.ui.h:31
msgid ""
"You are connected via roaming and may be charged for the data consumed by "
"this update."
msgstr ""
"Jste připojeni přes roaming a mohou vám být účtovány poplatky za data "
"stažená během této aktualizace."

#: ../data/gtkbuilder/UpdateManager.ui.h:32
msgid ""
"You may want to wait until you’re not using a mobile broadband connection."
msgstr ""

#: ../data/gtkbuilder/UpdateManager.ui.h:33
msgid "It’s safer to connect the computer to AC power before updating."
msgstr ""
"Před spuštěním aktualizace je bezpečnější připojit počítač k síťovému "
"napájení."

#: ../data/gtkbuilder/UpdateManager.ui.h:34
msgid "_Settings..."
msgstr "_Nastavení..."

#: ../data/gtkbuilder/UpdateManager.ui.h:35
#, fuzzy
msgid "_Install Now"
msgstr "Nainstalovat"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Je k dispozici nové vydání Ubuntu. Chcete přejít na vyšší verzi?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "Nepřecházet"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Ask Me Later"
msgstr "Dotázat se později"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "Yes, Upgrade Now"
msgstr "Ano, přejít nyní"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Odmítli jste přechod na nové Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
#, fuzzy
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Přejít na novější vydání můžete později spuštěním Správce aktualizací a "
"kliknutím na „Přejít na vyšší verzi“."

#: ../data/update-manager.desktop.in.h:2
msgid "Software Updates"
msgstr "Aktualizace softwaru"

#: ../data/update-manager.desktop.in.h:3
msgid "Show and install available updates"
msgstr "Zobrazit a nainstalovat dostupné aktualizace"

#: ../update-manager:66 ../update-manager-text:55 ../do-release-upgrade:51
msgid "Show version and exit"
msgstr "Zobrazit verzi a skončit"

#: ../update-manager:69
msgid "Directory that contains the data files"
msgstr "Složka, která obsahuje datové soubory"

#: ../update-manager:72
msgid "Check if a new Ubuntu release is available"
msgstr "Zkontrolovat, zda není k dispozici nové vydání Ubuntu"

#: ../update-manager:75 ../do-release-upgrade:54 ../check-new-release-gtk:186
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Zkontrolovat, zda je možné povýšit systém na nejnovější vývojové vydání"

#: ../update-manager:79
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr ""
"Povýšit systém za použití poslední navržené verze aktualizačního nástroje"

#: ../update-manager:86
msgid "Do not focus on map when starting"
msgstr "Při startu nezaměřovat na mapu"

#: ../update-manager:89
msgid "Try to run a dist-upgrade"
msgstr "Pokusit se povýšit systém"

#: ../update-manager:92
msgid "Do not check for updates when starting"
msgstr "Nekontrolovat aktualizace při spouštění"

#: ../update-manager:96 ../do-release-upgrade:70
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Otestovat aktualizaci se zabezpečeným prostředím aufs"

#: ../update-manager:116
msgid "Running partial upgrade"
msgstr "Probíhá částečné povýšení systému"

#: ../update-manager-text:59
msgid "Show description of the package instead of the changelog"
msgstr "Ukázat popis balíku namísto seznamu změn"

#: ../do-release-upgrade:58 ../check-new-release-gtk:190
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Zkuste povýšit na poslední vydání pomocí aktualizačního systému z $distro-"
"proposed"

#: ../do-release-upgrade:62
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Spustit ve speciálním režimu povýšení systému.\n"
"V současné době je podporováno „desktop“ pro systémy na osobních počítačích "
"a „server“ pro systémy na serverech."

#: ../do-release-upgrade:68
msgid "Run the specified frontend"
msgstr "Spustit určenou nádstavbu"

#: ../do-release-upgrade:73
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Zkontrolovat, jen pokud je k dispozici nové vydání distribuce a výsledek "
"oznámit přes kód ukončení"

#: ../do-release-upgrade:87
msgid "Checking for a new Ubuntu release"
msgstr "Probíhá pokus o nalezení nové verze Ubuntu"

#: ../do-release-upgrade:101
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Pro informace o přechodu na vyšší verzi prosím navštivte:\n"
"%(url)s\n"

#: ../do-release-upgrade:107
msgid "No new release found"
msgstr "Nenalezeno žádné nové vydání"

#: ../do-release-upgrade:119
#, c-format
msgid "New release '%s' available."
msgstr "Dostupné nové vydání „%s“."

#: ../do-release-upgrade:120
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Spusťte „do-release-upgrade“ pro povýšení systému na toto vydání."

#: ../check-new-release-gtk:101
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Přechod na Ubuntu %(version)s k dispozici"

#: ../check-new-release-gtk:143
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Odmítli jste přechod na Ubuntu %s"

#: ../check-new-release-gtk:196
msgid "Add debug output"
msgstr "Přidat ladící výstup"

#: ../ubuntu-support-status:91
msgid "Show unsupported packages on this machine"
msgstr "Ukázat balíky nepodporované na tomto stroji"

#: ../ubuntu-support-status:94
msgid "Show supported packages on this machine"
msgstr "Ukázat balíky podporované na tomto stroji"

#: ../ubuntu-support-status:97
msgid "Show all packages with their status"
msgstr "Ukázat všechny balíky s jejich stavy"

#: ../ubuntu-support-status:100
msgid "Show all packages in a list"
msgstr "Ukázat všechny balíky v seznamu"

#: ../ubuntu-support-status:142
#, c-format
msgid "Support status summary of '%s':"
msgstr "Souhrn stavu podpory '%s':"

#: ../ubuntu-support-status:145
msgid "You have %(num)s packages (%(percent).1f%%) supported until %(time)s"
msgstr "Máte %(num)s balíků  (%(percent).1f%%) podporovaných do %(time)s"

#: ../ubuntu-support-status:151
msgid ""
"You have %(num)s packages (%(percent).1f%%) that can not/no-longer be "
"downloaded"
msgstr "Máte %(num)s balíků (%(percent).1f%%), které již nelze stahovat."

#: ../ubuntu-support-status:154
msgid "You have %(num)s packages (%(percent).1f%%) that are unsupported"
msgstr "Máte %(num)s balíků (%(percent).1f%%), které jsou nepodporované"

#: ../ubuntu-support-status:162
msgid ""
"Run with --show-unsupported, --show-supported or --show-all to see more "
"details"
msgstr ""
"Spusťte s parametry  --show-unsupported, --show-supported nebo --show-all "
"pro více detailů"

#: ../ubuntu-support-status:166
msgid "No longer downloadable:"
msgstr "Již nelze stáhnout:"

#: ../ubuntu-support-status:169
msgid "Unsupported: "
msgstr "Nepodporováno: "

#: ../ubuntu-support-status:174
#, c-format
msgid "Supported until %s:"
msgstr "Podporováno do %s:"

#: ../ubuntu-support-status:183
msgid "Unsupported"
msgstr "Nepodporováno"

#. Why do we use %s here instead of $strings or {} format placeholders?
#. It's because we don't want to break existing translations.
#: ../janitor/plugincore/exceptions.py:42
#, python-format
msgid "Unimplemented method: %s"
msgstr "Neimplementovaná metoda: %s"

#: ../janitor/plugincore/core/file_cruft.py:41
msgid "A file on disk"
msgstr "soubor na disku"

#: ../janitor/plugincore/core/missing_package_cruft.py:39
msgid "Install missing package."
msgstr "Nainstalovat chybějící balíček."

#. 2012-06-08 BAW: i18n string; don't use {} or PEP 292.
#: ../janitor/plugincore/core/missing_package_cruft.py:49
#, python-format
msgid "Package %s should be installed."
msgstr "Balíček %s by měl být nainstalován."

#: ../janitor/plugincore/core/package_cruft.py:49
msgid ".deb package"
msgstr "balíček .deb"

#: ../janitor/plugincore/plugins/langpack_manual_plugin.py:46
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s musí být označeno jako ručně nainstalované."

#: ../janitor/plugincore/plugins/kdelibs4to5_plugin.py:49
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Pokud je nainstalován kdelibs4-dev, je při přechodu na vyšší verzi nutné "
"doinstalovat kdelibs5-dev. Více podrobností zjistíte na bugs.launchpad.net, "
"chyba #279621"

#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:44
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i zastaralých položek ve stavovém souboru"

#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:47
msgid "Obsolete entries in dpkg status"
msgstr "Zastaralé záznamy v dpkg status"

#. pragma: no cover
#: ../janitor/plugincore/plugins/dpkg_status_plugin.py:50
msgid "Obsolete dpkg status entries"
msgstr "Zastaralé záznamy v dpkg status"

#: ../janitor/plugincore/plugins/remove_lilo_plugin.py:42
msgid "Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Odstranit lilo, neboť grub je taktéž nainstalován. (Více informací v chybě "
"#314004.)"

#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore so a bug reporting process is being started."
#~ msgstr ""
#~ "Poté co byly aktualizovány informace o balíku, nezbytný balík '%s' nemůže "
#~ "být nalezen, proces hlášení chyby zahájen."

#~ msgid "<b><big>Upgrading Ubuntu to version 12.04</big></b>"
#~ msgstr "<b><big>Aktualizace Ubuntu na verzi 12.04</big></b>"

#~ msgid "%(count)s update has been selected."
#~ msgid_plural "%(count)s updates have been selected."
#~ msgstr[0] "Byla vybrána %(count)s aktualizace."
#~ msgstr[1] "Byly vybrány %(count)s aktualizace."
#~ msgstr[2] "Bylo vybráno %(count)s aktualizací."

#~ msgid "%(count_str)s %(download_str)s"
#~ msgstr "%(count_str)s %(download_str)s"

#~ msgid "Welcome to Ubuntu"
#~ msgstr "Vítejte v Ubuntu"

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released."
#~ msgstr "Tyto aktualizace byly vydány od doby vydání této verze Ubuntu."

#~ msgid "Software updates are available for this computer."
#~ msgstr "Pro tento počítač jsou dostupné aktualizace."

#~ msgid "Update Manager"
#~ msgstr "Správce aktualizací"

#~ msgid "Starting Update Manager"
#~ msgstr "Spouští se Správce aktualizací"

#~ msgid "You are connected via a wireless modem."
#~ msgstr "Jste připojeni přes bezdrátový modem."

#~ msgid "_Install Updates"
#~ msgstr "Na_instalovat aktualizace"

#~ msgid "Your system is up-to-date"
#~ msgstr "Váš systém je plně aktualizovaný"

#~ msgid ""
#~ "Fetching and installing the upgrade can take several hours. Once the "
#~ "download has finished, the process cannot be cancelled."
#~ msgstr ""
#~ "Stahování aktualizací a povyšování systému může trvat několik hodin. "
#~ "Jakmile stahování skončí, tak už tento proces nelze zrušit."

#~ msgid "Software updates are available for this computer"
#~ msgstr "Pro tento počítač jsou dostupné aktualizace softwaru"

#~ msgid "Checking for a new ubuntu release"
#~ msgstr "Kontroluje se nové vydání Ubuntu"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a systemdir from now until the next reboot are "
#~ "permanent."
#~ msgstr ""
#~ "Tato aktualizace běží v režimu zabezpečeného (testovacího) prostředí. "
#~ "Všechny změny budou zapsány do „%s“ a budou ztraceny při příštím "
#~ "restartu.\n"
#~ "\n"
#~ "*Žádné* změny zapsané do systémové složky od teď až do restartu nejsou "
#~ "trvalé."

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from the "
#~ "Administration menu later."
#~ msgstr ""
#~ "Pokud je nyní nechcete instalovat, zvolte později „Správce aktualizací“ z "
#~ "nabídky Správa."

#~ msgid "There are no updates to install"
#~ msgstr "Nejsou žádné aktualizace k instalaci"

#~ msgid "The update has already been downloaded, but not installed"
#~ msgid_plural "The updates have already been downloaded, but not installed"
#~ msgstr[0] "Aktualizace byla již stažena, ale nenainstalována"
#~ msgstr[1] "Aktualizace byly již staženy, ale nenainstalovány"
#~ msgstr[2] "Aktualizace byly již staženy, ale nenainstalovány"

#~ msgid ""
#~ "You will not get any further security fixes or critical updates. Please "
#~ "upgrade to a later version of Ubuntu Linux."
#~ msgstr ""
#~ "Nebudete dostávat žádné další bezpečností záplaty ani kritické "
#~ "aktualizace. Prosím přejděte na vyšší verzi Ubuntu Linuxu."

#~ msgid ""
#~ "If you don't want to install them now, choose \"Update Manager\" from "
#~ "Applications later."
#~ msgstr ""
#~ "Pokud je nechcete nyní instalovat, zvolte později z nabídky Aplikace "
#~ "\"Správce aktualizací\"."

#~ msgid "%.0f kB"
#~ msgstr "%.0f kB"

#~ msgid "0 kB"
#~ msgstr "0 kB"

#~ msgid "1 kB"
#~ msgstr "1 kB"

#~ msgid "Your graphics hardware may not be fully supported in Ubuntu 11.04."
#~ msgstr "Váš grafický hardware nemusí být plně podporován v Ubuntu 11.04."

#~ msgid ""
#~ "\n"
#~ "\n"
#~ "Please report this bug using the command 'ubuntu-bug update-manager' in a "
#~ "terminal and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report.\n"
#~ "%s"
#~ msgstr ""
#~ "\n"
#~ "\n"
#~ "Nahlaste prosím tuto chybu pomocí příkazu 'ubuntu-bug update-manager' v "
#~ "terminálu a přiložte soubory z /var/log/dist-upgrade/ do chybového "
#~ "hlášení.\n"
#~ "%s"

#~ msgid ""
#~ "The support in Ubuntu 11.04 for your intel graphics hardware is limited "
#~ "and you may encounter problems after the upgrade. Do you want to continue "
#~ "with the upgrade?"
#~ msgstr ""
#~ "Podpora vašeho grafického hardware intel v Ubuntu 11.04 je omezena a po "
#~ "upgradu se můžete setkat s problémy. Chcete pokračovat v upgradu?"

#~ msgid ""
#~ "Preparing the system for the upgrade failed. Please report this using the "
#~ "command 'ubuntu-bug update-manager' in a terminal and include the files "
#~ "in /var/log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Příprava systému na upgrade selhala. Nahlaste prosím tuto chybu pomocí "
#~ "příkazu 'ubuntu-bug update-manager' v terminálu a přiložte soubory z /var/"
#~ "log/dist-upgrade/ do chybového hlášení."

#~ msgid ""
#~ "Upgrading the repository information resulted in a invalid file. Please "
#~ "report this as a bug using the command 'ubuntu-bug update-manager' in a "
#~ "terminal."
#~ msgstr ""
#~ "Povýšení repozitáře selhalo. Prosím nahlaste chybu použitím příkazu "
#~ "'ubuntu-bug update-manager' v terminálu."

#~ msgid ""
#~ "The system was unable to get the prerequisites for the upgrade. The "
#~ "upgrade will abort now and restore the original system state.\n"
#~ "\n"
#~ "Please report this as a bug using the command 'ubuntu-bug update-manager' "
#~ "in a terminal and include the files in /var/log/dist-upgrade/ in the bug "
#~ "report."
#~ msgstr ""
#~ "Systém nebyl schopen získat nástroje pro povýšení systému. Upgrade bude "
#~ "přerušen a provede se obnova aktuálního stavu systému.\n"
#~ "\n"
#~ "Prosím nahlaste tuto chybu použitím příkazu 'ubuntu-bug update-manager' v "
#~ "terminálu a přiložte soubory ve /var/log/dist-upgrade do popisu chyby."

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released. If you don't want to install them now, choose \"Update Manager"
#~ "\" from Applications later."
#~ msgstr ""
#~ "Tyto aktualizace byly zveřejněny po vydání této verze Ubuntu. Pokud je "
#~ "nechcete nyní instalovat, zvolte později z nabídky Aplikace \"Správce "
#~ "aktualizací\"."

#~ msgid ""
#~ "These software updates have been issued since this version of Ubuntu was "
#~ "released. If you don't want to install them now, choose \"Update Manager"
#~ "\" from the Administration Menu later."
#~ msgstr ""
#~ "Tyto aktualizace byly zveřejněny po vydání této verze Ubuntu. Pokud je "
#~ "nechcete nyní instalovat, zvolte později z nabídky Správa \"Správce "
#~ "aktualizací\"."

#~ msgid ""
#~ "After your package information was updated the essential package '%s' can "
#~ "not be found anymore.\n"
#~ "This indicates a serious error, please report this bug using the command "
#~ "'ubuntu-bug update-manager' in a terminal and include the files in /var/"
#~ "log/dist-upgrade/ in the bug report."
#~ msgstr ""
#~ "Po aktualizaci informací o balících nebyl nalezen základní balík '%s'.\n"
#~ "Toto naznačuje závažný problém, nahlaste jej prosím pomocí příkazu "
#~ "'ubuntu-bug update-manager' a ke hlášení přiložte soubory ve /var/log/"
#~ "dist-upgrade/."
